### DISCUSSION TITLE
[Build for Windows] - Error on sqlite3.lib not found

### DISCUSSION THREAD

**Discussion by TristanPouliquen** [tauri-apps/tauri#6183] [Q&A] [answered]
Hello,

I am trying to build my first Tauri App, and I am using SQLite to store my data.

In my Github Actions, I am trying to build my app to bundle it for Windows, MacOs & Ubuntu.

I have an error when bundling on a Windows image, stating that it cannot find `sqlite3.lib`.

Do you know if there is a way for me to bundle it inside my executable? Or at least install it during my CI pipeline so the build completes ? I could tell my users to install SQLite3 before using my software even if it is not as user-friendly

I haven't found a template or example that tackles this issue

My repo is: https://github.com/TristanPouliquen/selection-poste-enm

I tried adding `sqlx` but to no avail.

Thanks in advance!

**Comment by FabianLars** [comment, marked as answer]
It's a bit annoying that diesel doesn't have this feature built-in if you ask me, but you can simulate it by adding this to your cargo.toml dependencies:
```toml
libsqlite3-sys = { version = ">=0.17.2, <0.26.0", features = ["bundled"] }
```
(the version range is taken from <https://github.com/diesel-rs/diesel/blob/v2.0.3/diesel/Cargo.toml> and you may have to update that on major diesel versions.

**Reply by TristanPouliquen** [reply]
Sweet, I will try it! Will it bundle correctly SQLite in the .exe for Windows or will my users still need to manually install SQLite before?

**Reply by FabianLars** [reply]
Unless diesel does some really weird stuff (afaik it doesn't), your users won't have to do anything.

**Reply by TristanPouliquen** [reply]
It builds! 🎉 

Thank you for your so swift answer to my problem!
